doc/io.xml

```diff
@@ -105,3 +105,80 @@ gap> ReadGenerators(file, 13);
     </Description>
   </ManSection>
 <#/GAPDoc>
+
+<#GAPDoc Label="IteratorFromMultiplicationTable">
+  <ManSection>
+    <Func Name = "IteratorFromMultiplicationTable" Arg = "filename"/>
+    <Returns>An iterator.</Returns>
+    <Description>
+      If <A>filename</A> is a string containing the name of a file created using
+      <Ref Func = "WriteMultiplicationTable"/>, then 
+      <C>IteratorFromMultiplicationTable</C> returns an iterator <C>iter</C>
+      such that <C>NextIterator(iter)</C> returns the next multiplication table
+      stored in the file <A>filename</A>.<P/>
+
+      This function is a convenient way of, for example, looping over a
+      collection of multiplication tables in a file without loading every
+      object in the file into memory. This might be useful if the file
+      contains more information than there is available memory.<P/>
+    </Description>
+  </ManSection>
+<#/GAPDoc>
+
+<#GAPDoc Label="WriteMultiplicationTable">
+  <ManSection>
+    <Func Name = "WriteMultiplicationTable" Arg = "filename, list[, append]"/>
+    <Returns><C>IO_OK</C> or <C>IO_ERROR</C>.</Returns>
+    <Description>
+      This function provides a method for writing collections of multiplication
+      tables to a file.  The resulting file can be further compressed using
+      <C>gzip</C> or <C>xz</C>.<P/>
+
+      The argument <A>list</A> should be a list of multiplication tables.<P/>
+
+      The argument <A>filename</A> should be a string containing the name of a
+      file where the entries in <A>list</A> will be written or an
+      <Package>IO</Package> package file object; see <Ref Func="IO_File"
+        BookName="io"/> and <Ref Func="IO_CompressedFile" BookName="io"/>.<P/>
+
+      If  the  optional  third argument <A>append</A> is given and equals
+      <C>"w"</C> or is not present, then the previous content of the file
+      is deleted and overwritten. If the optional third argument is <C>"a"</C>
+      or is not present, then <C>list</C> is appended to the file. This
+      function returns <Ref Var="IO_OK" BookName="io"/> if everything went well
+      or <Ref Var="IO_ERROR" BookName="io"/> if something went wrong.<P/>
+
+      <C>WriteMultiplicationTable</C> appends a line to the file
+      <A>filename</A> for every entry in <A>list</A>.<P/>
+      
+      The file <A>filename</A> can be read using 
```

Comment:
Maybe "The multiplication tables saved in <A>filename</A> can be recovered from the file using ..."